packages/grafana-ui/src/themes/GlobalStyles/elements.ts

```diff
@@ -40,11 +40,6 @@ export function getElementStyles(theme: GrafanaTheme2) {
       position: 'unset',
       color: theme.colors.text.primary,
       backgroundColor: theme.colors.background.canvas,
-      // react select tries prevent scrolling by setting overflow/padding-right on the body
-      // Need type assertion here due to the use of !important
-      // see https://github.com/frenic/csstype/issues/114#issuecomment-697201978
-      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
-      overflowY: 'auto !important' as 'auto',
```

Comment:
@svennergr it's important that the page body scroll when the side pane is not open (so you can take full page screenshots and other things), is this still true with this change?

Reply:
Oh, it's not. Let me take a look, thanks!

Reply:
This is now implemented in 08325b527b2f13ad5144b253948b787978bdcc4d


https://github.com/user-attachments/assets/9fabcb14-416d-49ce-890c-a22990965425